src/components/Profile/components/Identification/index.js

```diff
@@ -718,25 +715,16 @@ const Identification = ({ profile, myProf, network, createSnackbar }) => {
                     userProfile.Title !== '' &&
                     userProfile.PersonType === 'fac'
                       ? // If the user has a title
-                        hasNickName
-                        ? // If the user has a title and a nickname
-                          userProfile.Title +
-                          ' ' +
-                          userProfile.fullName +
-                          ' (' +
-                          userProfile.NickName +
-                          ')'
-                        : // If the user has a title and no nickname
-                          userProfile.Title + ' ' + userProfile.fullName
+                        `${userProfile.Title} ${userProfile.FirstName}${
+                            hasNickname ? ` (${userProfile.NickName})` : ''
+                        } ${userProfile.LastName}${
+                            hasMaidenName ? ` (${userProfile.MaidenName})` : ''}`
                       : // If the user doesn't have a title
-                      hasNickName
-                      ? // If the user doesn't have a title but has a nickname
-                        userProfile.fullName + ' (' + userProfile.NickName + ')'
-                      : // If the user doesn't have a title or a nickname
-                        userProfile.fullName}
-                    {/* {hasNickName
-                        ? userProfile.fullName + ' (' + userProfile.NickName + ')'
-                        : userProfile.fullName} */}
+                      `${userProfile.FirstName}${
+                            hasNickname ? ` (${userProfile.NickName})` : ''
+                        } ${userProfile.LastName}${
+                            hasMaidenName ? ` (${userProfile.MaidenName})` : ''}`
+                    }
```

Comment:
This might be overly picky, but I do see an opportunity to reduce code duplication even further, by putting the logic for the title inside the template string, you only need one template string instead of two:
```javascript
                    `${
                          userProfile.Title && userProfile.PersonType === 'fac' ? `${userProfile.Title} ` : ''
                     }${userProfile.FirstName}${
                            hasNickname ? ` (${userProfile.NickName})` : ''
                     } ${userProfile.LastName}${
                            hasMaidenName ? ` (${userProfile.MaidenName})` : ''}`
                    }
```

Reply:
This is very helpful! I just committed this to this branch. Can you approve it again?